src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.js

```diff
@@ -24,6 +25,11 @@ import PaymentList from "./PaymentList";
 export default function PackageReview() {
   const params = useParams();
   const packageId = Number(params.packageId);
+
+  const location = useLocation();
+  const queryParams = queryString.parse(location.search);
+  const { returnUrl } = queryParams;
```

Comment:
Should we validate that the return url is a valid one, so we don't show the button if it is not valid?

Reply:
Done.  Added check to only show button if returnUrl is valid